src/SignalR/server/SignalR/test/MapSignalRTests.cs

```diff
@@ -101,6 +138,49 @@ public void MapHubFindsMultipleAuthAttributesOnDoubleAuthHub()
             Assert.Equal(2, authCount);
         }
 
+        [Fact]
+        public void MapHubEndPointRoutingFindsAttributesOnHub()
+        {
+            var authCount = 0;
+            using (var host = BuildWebHostWithEndPointRouting(routes => routes.MapHub<AuthHub>("/path", options =>
+            {
+                authCount += options.AuthorizationData.Count;
+            })))
+            {
+                host.Start();
+
+                var dataSource = host.Services.GetRequiredService<EndpointDataSource>();
```

Comment:
Did we want to create a name data source? `SignalREndpointDataSource`

Reply:
Nope. No need.